File: MainApp/src/main/java/com/sleticalboy/dailywork/ui/activity/ScrollerActivity.java
package com.sleticalboy.dailywork.ui.activity;

import android.view.View;
import android.widget.Button;
import android.widget.LinearLayout;

import com.sleticalboy.dailywork.R;
import com.sleticalboy.dailywork.base.BaseActivity;

/**
 * Created on 18-6-10.
 *
 * @author sleticalboy
 * @description Scroller 使用练习
 */
public class ScrollerActivity extends BaseActivity {

    private LinearLayout layout;

    @Override
    protected int layoutResId() {
        return R.layout.activity_scroller;
    }

    @Override
    protected void initView() {
        layout = findViewById(R.id.layout);
        final Button scrollToBtn = findViewById(R.id.scroll_to_btn);
        final Button scrollByBtn = findViewById(R.id.scroll_by_btn);
        scrollToBtn.setOnClickListener(v -> layout.scrollTo(-60, -100));
        scrollByBtn.setOnClickListener(v -> layout.scrollBy(-60, -100));
    }

    @Override
    protected void initData() {
    }
}

File: app/src/main/java/com/binlee/learning/csv/ScrollerActivity.kt
package com.binlee.learning.csv

import android.view.View
import com.binlee.learning.base.BaseActivity
import com.binlee.learning.databinding.ActivityScrollerBinding

/**
 * Created on 18-6-10.
 *
 * @author leebin
 * @description Scroller 使用练习
 */
class ScrollerActivity : BaseActivity() {

  private var bind: ActivityScrollerBinding? = null

  override fun layout(): View {
    // R.layout.activity_scroller
    bind = ActivityScrollerBinding.inflate(layoutInflater)
    return bind!!.root
  }

  override fun initView() {
    bind!!.scrollToBtn.setOnClickListener { bind!!.layout.scrollTo(-60, -100) }
    bind!!.scrollByBtn.setOnClickListener { bind!!.layout.scrollBy(-60, -100) }
  }

  override fun initData() {}
}